offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -509,5 +521,6 @@ class CreatePersonView extends FormLayout implements Resettable{
     void reset() {
         createPersonViewModel.reset()
         clearAllFields()
+        initDefaultValues()
```

Comment:
This is not needed if the ViewModel reset is implemented correctly.

Reply:
@KochTobi This is needed since the clearallFields method clears the values set in the reset method of the viewModel.